python/paddleserver/paddleserver/model.py

```diff
@@ -59,17 +62,17 @@ def get_model_files(ext: str) -> str:
         self.ready = True
         return self.ready
 
-    def predict(self, payload: Dict, headers: Dict[str, str] = None) -> Dict:
-        instances = payload["instances"]
+    def predict(self, payload: Union[Dict, InferRequest], headers: Dict[str, str] = None) -> Union[Dict, InferResponse]:
         try:
-            inputs = np.array(instances, dtype='float32')
+            instance = get_predict_input(payload)
+            np_array_input = np.array(instance, dtype='float32')
```

Comment:
for v2 protocol the instance is already np array which is the return type from `get_predict_input`